albumentations/core/bbox_utils.py

```diff
@@ -118,7 +119,7 @@ def check(self, data: Sequence, rows: int, cols: int) -> None:
         check_bboxes(data)
 
     def convert_from_albumentations(self, data: Sequence, rows: int, cols: int) -> List[BoxType]:
-        return convert_bboxes_from_albumentations(data, self.params.format, rows, cols, check_validity=True)
+        return convert_bboxes_from_albumentations(data, self.params.format, rows, cols, check_validity=True, clamp_bbox=self.params.clamp_bbox)
 
     def convert_to_albumentations(self, data: Sequence[BoxType], rows: int, cols: int) -> List[BoxType]:
         return convert_bboxes_to_albumentations(data, self.params.format, rows, cols, check_validity=True)
```

Comment:
You added this parameter to wrong function. You changed `to` function, but added param to `from` function. Please add small test to check that this functionality works coreectly